facturx/facturx.py

```diff
@@ -148,23 +148,23 @@ def write_xml(self, path):
             f.write(self.xml_str)
 
     def __make_dict(self, flavor):
-        dict_data = xml_flavor.FIELDS
+        fields_data = xml_flavor.FIELDS
 
         required_list = None
```

Comment:
Building this `required_list` is useless because you are looping over the same dict again later.